server/storage/sqldb.go

```diff
@@ -241,6 +289,7 @@ func (db *SQLStorage) Delete(gun data.GUN) error {
 		}
 		// if there weren't actually any records for the GUN, don't write
 		// a deletion change record.
+		// don't write a change if this isn't published yet
```

Comment:
heh, yes this is a straggling comment - originally I scoped the delete but realized, as you point out, that when we delete a GUN we want to delete all staged copies as well